lib/Service/NoteUtil.php

```diff
@@ -186,7 +186,12 @@ public function getOrCreateFolder(string $path, bool $create = true) : Folder {
 	public function getNotesFolderUserPath(string $userId): ?string {
 		/** @psalm-suppress MissingDependency */
 		$userFolder = $this->getRoot()->getUserFolder($userId);
-		$nodesFolder = $this->getOrCreateNotesFolder($userId, false);
+		try {
+			$nodesFolder = $this->getOrCreateNotesFolder($userId, false);
+		} catch (NotesFolderException $e) {
+			$this->util->logger->error("Failed to get notes folder for user $userId: " . $e->getMessage());
```

Comment:
Not always an error. New users, or haven't used `Notes` it is fine that it is not present. So, level could be downgraded.